sdk/servicebus/service-bus/src/receivers/receiverCommon.ts

```diff
@@ -213,12 +213,24 @@ export function deadLetterMessage(
     retryOptions,
   };
 
-  return settleMessage(
-    message,
-    DispositionType.deadletter,
-    context,
-    entityPath,
-    dispositionStatusOptions
+  const tracingContext = extractSpanContextFromServiceBusMessage(message);
+  const spanLinks: TracingSpanLink[] = tracingContext ? [{ tracingContext }] : [];
+
+  return tracingClient.withSpan(
+    "ServiceBusReceiver.deadLetter",
```

Comment:
nice! do we have similar ones for other settlement operations?

Reply:
yes, we have spans for `complete`, `abandon`, and `defer` too.